### DISCUSSION TITLE
Adding charge cost manually?

### DISCUSSION THREAD

**Discussion by redmamoth** [teslamate-org/teslamate#3269] [General]
Apparently, TeslaMate has the functionality in the charges dashboard to manually add charge costs, you can click on a '-' in the Cost column and the link takes you to **<'Web App' URL from TeslaMate settings>/charge-cost/nn**.  

This link isn't working for me, i just reloads HASS inside itself and dumps me on the TeslaMate homepage.  Do I need to set the Web App URL to something special for it to work?  Currently I have it set to: https://hass.REDACTED.duckdns.org/REDACTED_teslamate/ingress

**Comment by cwanja** [comment]
What have you set to the URL in TeslaMate settings? Review that and set it accordingly. It uses a database configuration to build that URL.
